xml/IATIOrganisationIdentifier.xml

```diff
@@ -38,14 +38,26 @@
             <url>http://www.umn.org.np/</url>
         </codelist-item>
         <codelist-item>
-            <code>XI-IATI-CWSEC</code>
+            <code>XI-IATI-ADVZ</code>
             <name>
-                <narrative>The Commonwealth Secretariat</narrative>
+                <narrative>Agência de Desenvolvimento do Vale do Zambeze</narrative>
             </name>
             <description>
-                <narrative>The Commonwealth is a voluntary association of 52 independent and equal sovereign states. Its guiding principles are contained in their Commonwealth Charter.</narrative>
+                <narrative>Description:
+                The Agência do Zambeze is a public-sector agency under the Ministry of Economics and Finance in Mozambique. The mission of the Agência do Zambeze is to promote the socio-economic and sustainable development of the Lower Zambezi River Basin.</narrative>
             </description>
-            <url>http://thecommonwealth.org/</url>
+            <url>http://www.inm.gov.mz/?q=pt-pt/ag%C3%AAncia-de-desenvolvimento-do-vale-do-zambeze</url>
+        </codelist-item>
+        <codelist-item>
+            <code>XI-IATI-AIAS</code>
+            <name>
+                <narrative>Administração de Infra-Estruturas de Águas e Saneamento</narrative>
+            </name>
+            <description>
+                <narrative>Description:
+                AIAS manages the property of the secondary public systems for water supply and for the residual water drainage public systems in Mozambique promoting their autonomous efficient and financially feasible operational management through assignment to private operators or other third party entities.</narrative>
+            </description>
+            <url>http://www.aias.gov.mz/1</url>
```

Comment:
Worth checking the URL on this one - according to [Discuss post](https://discuss.iatistandard.org/t/approved-create-org-identifier-xi-iati-aias-for-the-administracao-de-infra-estruturas-de-aguas-e-saneamento-mozambique/976), it should be `http://www.aias.gov.mz/`